modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -25,10 +25,15 @@ An API fragment is a RAML document that has a version and an identifier, but is
 
```

Comment:
I think we should change the title, now the fragments can be RAML or OAS.
> Create and Publish a RAML API Fragment in the Text Editor in API Designer
> You can create API fragments directly in RAML with the help of code suggestions that appear within the text editor.

change to:
> Create and Publish an API Fragment in the Text Editor in API Designer
> You can create API fragments directly in RAML 1.0, OAS 3.0, or JSON schema with the help of code suggestions that appear within the text editor.

And in the section **About This Task** we could add something for the OAS Components:

> An OAS 3.0 component is an OAS document that has a version and an identifier but is not itself a complete OAS specification. It should contain only the reusable “components" object for different aspects of the OAS. All objects defined within the components object will have no effect on the API unless explicitly referenced from properties outside the components object.